### ISSUE TITLE
Incompatible genomes when loading VCF file

### ISSUE THREAD

**Issue by sunjh22** [jpuntomarcos/CNVfilteR#9]
Hi @jpuntomarcos ,

I am using CNVfilteR to filter false-positive CNVs based on BAF. I successfully loaded CNV region file, but when loading vcf file by `loadVCFs()`, an error occured: `Error in mergeNamedAtomicVectors(genome(x), genome(y), what = c("sequence",  : 
 sequences chr1, chr2, chr3, chr4, chr5, chr6, chr7, chr8, chr9, chr10, chr11, chr12, chr13, chr14, chr15, chr16, chr17, chr18, chr19, chr20, chr21, chr22, chrX, chrY have incompatible genomes: - in 'x': hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19, hg19 - in 'y': hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38, hg38`.
In the whole process, I used same reference genome from UCSC (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/analysisSet/). When loading CNV file, I used `genome='hg38'`.
If I use `genome='hg19'`, another `incompatible seqlengths` error happened.
I also attached a sample vcf file for your reference.
[sample1.snp.vcf.gz](https://github.com/jpuntomarcos/CNVfilteR/files/9791489/sample1.snp.vcf.gz)

Do you have any idea of what's happening? Thank you.

**Comment by jpuntomarcos** [comment]
Hi!

Are you setting genome='hg38' in both loadVCFs() and loadCNVcalls() functions? If so, please, provide me a minimum example that triggers the error so I can check what is happening :)

**Comment by sunjh22** [comment]
Hi jpuntomarcos,

Thanks for your reponse, it worked when I also set `genome=hg38` in `loadVCFs()`! Sorry for asking this stupid question... But I suggest also pointing that out in the package manual (I referred to this one: https://bioconductor.org/packages/devel/bioc/vignettes/CNVfilteR/inst/doc/CNVfilteR.html), so everyone knows that `genome=` is also an option in `loadVCFs()` function as in `loadCNVcalls()` function. Have a good day.
